x/subspaces/keeper/msg_server.go

```diff
@@ -831,3 +832,44 @@ func (k msgServer) SetUserPermissions(goCtx context.Context, msg *types.MsgSetUs
 
 	return &types.MsgSetUserPermissionsResponse{}, nil
 }
+
+// UpdateSubspaceFeeTokens defines a rpc method for MsgUpdateSubspaceFeeTokens
+func (k msgServer) UpdateSubspaceFeeTokens(goCtx context.Context, msg *types.MsgUpdateSubspaceFeeTokens) (*types.MsgUpdateSubspaceFeeTokensResponse, error) {
+	authority := k.authority
+	if authority != msg.Authority {
+		return nil, errors.Wrapf(govtypes.ErrInvalidSigner, "invalid authority; expected %s, got %s", authority, msg.Authority)
+	}
```

Comment:
No need to delcare a variable here if we're not going to use it anywhere outside this check
```suggestion
	if msg.Authority != k.authority {
		return nil, errors.Wrapf(govtypes.ErrInvalidSigner, "invalid authority; expected %s, got %s", k.authority, msg.Authority)
	}
```